src/main/java/jerome/tasklist/Task.java

```diff
@@ -0,0 +1,85 @@
+package jerome.tasklist;
+
+/**
+ * Represents a task with a description and completion status.
+ */
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    /**
+     * Constructs a Task object with the given description and completion status.
+     *
+     * @param description the description of the task.
+     * @param isDone the completion status of the task.
+     */
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+
+    /**
+     * Returns the status icon of the task.
+     * The status icon represents the completion status of the task.
+     * If the task is marked as done, the status icon will be "X".
+     * Otherwise, the status icon will be a space character.
+     *
+     * @return the status icon of the task ("X" if done, " " if not done)
+     */
```

Comment:
```suggestion
    /**
     * Returns the status icon of the task.
     * The status icon represents the completion status of the task.
     *
     * @return the status icon of the task ("X" if done, " " if not done)
     */
```

Can consider omitting the 2 lines as just the @return line is very clear